src/detection/GaussianPsf.cc

```diff
@@ -140,7 +140,7 @@ std::shared_ptr<GaussianPsf::Image> GaussianPsf::doComputeKernelImage(lsst::geom
             sum += row[xIndex] = std::exp(-0.5 * (x * x + y * y) / (_sigma * _sigma));
         }
     }
-    array.asEigen() /= sum;
+    ndarray::asEigenMatrix(array) /= sum;
```

Comment:
This looks a bit weird. How is `ndarray::asEigenMatrix` an lvalue? I found [the definition](https://github.com/ndarray/ndarray/pull/70/files#diff-c8d8179e4c56ee4ba4f1f314201ac543R256), but the return value doesn't look like a reference.

Reply:
Again, @TallJimbo may be able to explain it better, but I believe the fact that it is an `Eigen::Map` means it can be read or written. It's just a view into `array` data.

Reply:
Correct.  `Eigen::Map` is basically a "smart reference", something C++ is notoriously bad at providing intuitive syntax for.